pycbc/filter/matchedfilter.py

```diff
@@ -1247,7 +1247,7 @@ def matched_filter_core(template, data, psd=None, low_frequency_cutoff=None,
         h_norm = sigmasq(htilde, psd, low_frequency_cutoff, high_frequency_cutoff)
 
     norm = (4.0 * stilde.delta_f) / sqrt( h_norm)
-    delta_t = 1.0 / (N * stilde.delta_f)
+    delta_t = 1.0 / round(N * stilde.delta_f)
```

Comment:
Was this the source of the issue with the SNR series at the end? @ahnitz are you ok with this change in the matched filter core?

Reply:
@bhooshan-gadre Would using data.delta_t work in stead?

Reply:
@ahnitz data.delta_t is working. I will change it so to
```suggestion
    delta_t = stilde.delta_t
```